telegram/ext/dispatcher.py

```diff
@@ -593,6 +565,8 @@ def add_error_handler(self, callback, run_async=False):
         Note:
             See https://git.io/fxJuV for more info about switching to context based API.
         """
+        if callback in self.error_handlers:
+            raise ValueError('The callback is already registered as error handler.')
```

Comment:
I say just do `return None`.
You can issue a debug log if you feel that you must.